src/adjtrans.jl

```diff
@@ -92,11 +94,14 @@ function *(op :: ConjugateLinearOperator, v :: AbstractVector)
   return conj.(p * v)
```

Comment:
This line is buggy too. It should be `conj.(p * conj.(v))`.

Reply:
Fixed and added a test.